src/httpserver/handlers/handler.clj

```diff
@@ -1,24 +1,27 @@
 (ns httpserver.handlers.handler
-  (:require [httpserver.router :as router]))
+  (:require [httpserver.routing.router :as router]))
 
 (import '(com.td.HttpServer HttpResponse))
 (import '(com.td.HttpServer IHandler))
 (import '(com.td.HttpServer HttpRequest))
 
 (defn build-request [httpRequest]
-  (hash-map :path (.path httpRequest) :method (.method httpRequest) :headers (.headers httpRequest) :body (.body httpRequest)))
+  {:path (.path httpRequest) :method (.method httpRequest) :headers (.headers httpRequest) :body (.body httpRequest)})
```

Comment:
Since commas are optional in Clojure, you'll often see a long line like this broken into newlines for key/value pairs, i.e.
```clojure
{:path (.path httpRequest)
 :method (.method httpRequest)
 :headers (.headers httpRequest)
 :body (.body httpRequest)})
```
Just helps for readability